__tests__/app.test.js

```diff
@@ -70,17 +69,79 @@ describe('GET /api/reviews/:review_id', () => {
         return request(app)
         .get('/api/reviews/test')
         .expect(400)
-        .then(({ body }) => {
-            expect(body.msg).toBe('Bad request');
+        .then(({ body: { msg } }) => {
+            expect(msg).toBe('Bad request');
         });
     });
 
     test('404: should respond with "Route not found" if passed a valid number', () => {
         return request(app)
         .get('/api/reviews/100')
         .expect(404)
-        .then(({ body }) => {
-            expect(body.msg).toBe('Route not found');
+        .then(({ body: { msg }}) => {
+            expect(msg).toBe('No review found for review_id: 100');
+        });
+    });
+});
+
+describe('PATCH: /api/reviews/:review_id', () => {
+    test('200: should update the votes property within the selected review', () => {
+        const review_id = 2;
+        const userVote = { inc_votes: 10 };
+
+        return request(app)
+            .patch(`/api/reviews/${review_id}`)
+            .send(userVote)
+            .expect(200)
+            .then(({ body: { review }}) => { 
+                expect(review).toEqual({
+                    review_id: review_id,
+                    title: 'Jenga',
+                    designer: 'Leslie Scott',
+                    owner: 'philippaclaire9',
+                    review_img_url:
+                    'https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png',
+                    review_body: 'Fiddly fun for all the family',
+                    category: 'dexterity',
+                    created_at: '2021-01-18T10:01:41.251Z',
+                    votes: 15 
+                })
+            });
+    });
+
+    test('400: should respond with "Bad request" if not passed an integer through endpoint', () => {
+        const userVote = { inc_votes: 10 };
+
+        return request(app)
+        .patch('/api/reviews/test')
+        .send(userVote)
+        .expect(400)
+        .then(({ body:{ msg } }) => {
+            expect(msg).toBe('Bad request');
+        });
+    });
+
+    test('400: should respond with "Bad request" if user does not pass integer for inc_votes', () => {
+        const userVote = { inc_votes: 'hello' };
+
+        return request(app)
+        .patch('/api/reviews/2')
+        .send(userVote)
+        .expect(400)
+        .then(({ body: { msg } }) => {
+            expect(msg).toBe('Bad request');
+        });
+    });
+
+    test('404: should respond with "Route not found" if passed a valid number', () => {
+        const userVote = { inc_votes: 10 };
+
+        return request(app)
+        .patch('/api/reviews/100')
+        .send(userVote)
+        .expect(404)
+        .then(({ body: { msg }}) => {
+            expect(msg).toBe('No review found for review_id: 100');
```

Comment:
Good so far Karly. Can we put in a test to consider if the client sends an empty object for the body please...